pkgs/by-name/ov/overlayed/webui.nix

```
@@ -0,0 +1,43 @@
+{
+  src,
+  meta,
+  version,
+  lib,
+  stdenv,
+  nodejs,
+  pnpm,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  inherit version src meta;
+  pname = "overlayed-webui";
+
+  pnpmDeps = pnpm.fetchDeps {
+    src = "${finalAttrs.src}/apps/desktop";
+    version = finalAttrs.version;
+    pname = finalAttrs.pname;
+    hash = "sha256-FBJrVXTgKh8Op7TiZjA/i2LY2OOIKmqGR+GwGUecqVI=";
+  };
```

`pnpm-lock.yaml` is not located inside `apps/desktop`.
This only seemed to work because you forgot to invalidate the hash after changing the inputs.

Also, currently the `version` doesn't seem to affect the output name of the FOD.
(Though that might change in the future, so you can also inherit the version if you want to)

```suggestion
  pnpmDeps = pnpm.fetchDeps {
    inherit (finalAttrs) pname src;
    hash = "sha256-FBJrVXTgKh8Op7TiZjA/i2LY2OOIKmqGR+GwGUecqVI=";
  };
```